src/main/scala/nl.knaw.dans.easy.sword2/DepositProperties.scala

```diff
@@ -142,6 +142,11 @@ class DepositProperties(depositId: DepositId, depositorId: Option[String] = None
       .map(_.toString)
   }
 
+  def getUrn: Option[String] = {
```

Comment:
Did you create an issue to add this to https://github.com/DANS-KNAW/easy-deposit-properties

Reply:
good one. the URN:NBN is already in the deposit.properties, so on that front we are set